### DISCUSSION TITLE
How to replace the interface on /chat page from /api to /v1?

### DISCUSSION THREAD

**Discussion by X-SZM** [langgenius/dify#7535] [Help] [answered]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

How to replace the interface on /chat page from /api to /v1?

### 2. Additional context or comments

![image](https://github.com/user-attachments/assets/c06d8311-749a-4aa7-8101-364bf6b20cb3)
![image](https://github.com/user-attachments/assets/3d8fc0ad-c73e-4229-915a-b02f7c09d95b)

**Comment by crazywoola** [comment, marked as answer]
See this docs https://docs.dify.ai/getting-started/install-self-hosted/environments -> `SERVICE_API_URL `
